kurtosis-devnet/op-program-svc/README.md

```diff
@@ -0,0 +1,10 @@
+# Trigger new build:
+
+```
+$ curl -X POST -H "Content-Type: multipart/form-data" \
+    -F "files[]=@rollup-2151908.json" \
+    -F "files[]=@rollup-2151909.json" \
+    -F "files[]=@genesis-2151908.json" \
+    -F "files[]=@genesis-2151909.json" \
```

Comment:
nit: I neglected to mention this, but the interop program also needs a customized dependency set for the devnet. So:
```suggestion
    -F "files[]=@genesis-2151909.json" \
    -F "files[]=@depsets.json"
```
The depset isn't needed pre-interop.